tests/sqlite3/test-sqlite3.cpp

```diff
@@ -105,6 +105,35 @@ TEST_CASE("SQLite blob", "[sqlite][blob]")
     }
 }
 
+TEST_CASE("SQLite blob on a rowset", "[sqlite][blob][rowset]")
+{
+    soci::session sql(backEnd, connectString);
+
+    blob_table_creator tableCreator(sql);
+
+    char buf[] = "abcdefghijklmnopqrstuvwxyz";
+
+
+    {
+        blob b(sql);
+        b.write(0, buf, sizeof(buf));
+        sql << "insert into soci_test(id, img) values(1, ?)", use(b);
+        sql << "insert into soci_test(id, img) values(2, ?)", use(b);
+        sql << "insert into soci_test(id, img) values(3, ?)", use(b);
+        sql << "insert into soci_test(id, img) values(4, ?)", use(b);
+
+    }
+    {
+        rowset<row> rs = (sql.prepare << "select * from soci_test");
+        for(rowset<row>::iterator rsit = rs.begin(); rsit != rs.end(); rsit++)
+        {
+            // row &r = *rsit;
+            // soci::blob b = r.get<soci::blob>("img");
+            // CHECK(b.get_len() == sizeof(buf));
```

Comment:
Sorry, why is this commented out? The test doesn't seem to really test much as currently written.

Reply:
My goal was not really testing here, just check if it was still compiling for VS.

I am sorry if it was not clear enough.